pkg/util/util.go

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package util
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/k8sclient"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"math/rand"
+	"os"
+)
+
+// GetEnvValue looks for env variables in Operator pod to configure Code Ready deployments
+// with things like db users, passwords and deployment options in general. Envs are set in
+// a ConfigMap at deploy/config.yaml. Find more details on deployment options in README.md
+func GetEnv(key, defaultValue string) string {
+	value := os.Getenv(key)
+	if len(value) == 0 {
+		return defaultValue
+	}
+	return value
+}
+
+func GetEnvBool(key string, defaultValue bool) bool {
+	value := os.Getenv(key)
+	if len(value) == 0 {
+		return defaultValue
+	}
+	if value == "true" {
+		return true
+	}
+	return false
+}
+
+func GeneratePasswd(stringLength int) (passwd string) {
+	chars := []rune("ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
```

Comment:
I believe that there must be string generation utils in golang stdlib, but as an initial solution it is also OK